apps/allocations/app/components/Card/Account.js

```diff
@@ -41,56 +39,64 @@ const Account = ({
           </ContextMenuItem>
         </ContextMenu>
       </MenuContainer>
-      <IconContainer />
-      <TitleContainer>
-        <CardTitle>{description}</CardTitle>
-        <CardAddress>
-          <LocalIdentityBadge
-            networkType={type}
-            entity={proxy}
-            shorten={true}
-          />
-        </CardAddress>
-      </TitleContainer>
+      <CardTitle>{name}</CardTitle>
       <StatsContainer>
         <StyledStats>
-          <Text smallcaps color={theme.textSecondary}>
-            Balance
-          </Text>
-          <StatsValue>
-            {' ' + BigNumber(balance)
-              .div(ETH_DECIMALS)
-              .dp(3)
+          <StatsValueBig>
+            {displayCurrency(BigNumber(amount))}
+            <Text>{' ' + currency + ' per period'}</Text>
+          </StatsValueBig>
+          <StatsValueBig css={{ paddingTop: '24px' }}>
+            <ProgressBar
+              color="#1DD9D5"
```

Comment:
Let's use named colors from `theme`, if we can.

Reply:
This color doesn't exist in the `theme`. Already asked Javi.